packages/lesswrong/components/common/FormatDate.tsx

```diff
@@ -14,10 +14,38 @@ const formatRelative = (
   now: Date,
   includeAgo?: boolean,
 ): string => {
-  const formatted = moment(date).from(now);
+  //const formatted = formatRelativeMoment(new Date(date), now);
+  const formatted = formatRelativeFast(new Date(date), now);
   return includeAgo && formatted !== "now" ? formatted + " ago" : formatted;
 }
 
+const formatRelativeMoment = (date: Date, now: Date): string => {
+  return moment(date).from(now);
+}
+
+const formatRelativeFast = (date: Date, now: Date): string => {
+  const msApart = Math.abs(now.getTime() - date.getTime());
+  const secondsApart = msApart / 1000;
+  if (secondsApart < 44) {
+    return "now";
+  } else if (secondsApart < 45*60) {
+    const minutes = Math.round(secondsApart/60.0);
+    return `${minutes}m`;
+  } else if (secondsApart < 22*60*60) {
+    const hours = Math.round(secondsApart/(60.0*60.0));
+    return `${hours}h`;
```

Comment:
I would prefer minutes went up to 59 and hours went up to 23 (and maybe months up to 11 too), seeing as they are unambiguous

Reply:
The thresholds are copied from moment's default config, such that a unit test (after I write it) should be able to give the same timestamps to this function and to moment and get the same result. I don't have string opinions about the exact threshold; one of the benefits of doing it this way is that the thresholds are now in a place where we see them, rather than inside a library.